src/map/initMap.ts

```diff
@@ -43,6 +43,16 @@ export const initMap = (container: HTMLElement) => {
 
   map.touchZoomRotate.disableRotation();
 
+  if (get(params)?.geoLock) {
+    if (get(params).geoLock === "oa") {
+      // 9 seems a sensible compromise min zoom for urban and rural areas when OA is geolocked
+      map.setMinZoom(9);
+    } else {
+      const minZoomForGeoLock = layers.find((l) => l.name === get(params).geoLock).minZoom;
+      map.setMinZoom(minZoomForGeoLock);
+    }
+  }
+
```

Comment:
Since this only grabs the value at the time the map is created, it isn't changeable in the future? Might not be a problem for embeds.